### DISCUSSION TITLE
Can’t Sign in on the Neverforum

### DISCUSSION THREAD

**Discussion by KodeBreaker-Has-SWAG** [Neverball/neverball#260] [General]
There’s been an issue for around a month where users can’t sign in on the Neverforum. What happens is when I login into my account a message shows up saying I need to enable cookies for the site to login; even though I always have them enabled. I tried everything and still have the message popping up preventing me to login. I thought I was the only one with the problem so I went onto the Neverball Discord about the issue and everyone else on the server is having the same problem.

**Comment by parasti** [comment]
That sucks. Vidski (the person who owns the server) doesn't seem to be around much. I've tried to offer help, but no luck so far.

**Comment by SVNMaster** [comment]
Same. I can't also sign in

**Comment by clementtttttttt** [comment]
rip neverforum, it's essentially dead now

**Comment by KodeBreaker-Has-SWAG** [comment]
I know this is pretty late saying this, but to everyone who wants to continue to interact with the Neverball community can join the Neverball Discord server.

A good chunk of neverforum users are the server, here is the link:
https://discord.gg/HWp2kVQA

**Comment by AFCMS** [comment]
> I know this is pretty late saying this, but to everyone who wants to continue to interact with the Neverball community can join the Neverball Discord server.
> 
> A good chunk of neverforum users are the server, here is the link: https://discord.gg/HWp2kVQA

The invite link is expired

**Comment by qwertychouskie** [comment]
@AFCMS Here's a link that won't expire: https://discord.gg/U6dKHdnq8D

**Comment by Adacrop-The-Great** [comment]
Now the forum is inaccessible. RIP to years of Neverball history...

**Reply by clementtttttttt** [reply]
i hope someone made backups of the site on archive.org

**Reply by Adacrop-The-Great** [reply]
I went to the Wayback Machine today to access old versions of the Neverforum, but only the index is accessible. Whenever I click on a category, such as "Level Contributions", a message appears saying "neverforum.com’s server IP address could not be found".

**Reply by qwertychouskie** [reply]
If you use a tad older snapshot it works https://web.archive.org/web/20210513225642/http://neverforum.com/fmpbo/index.php

**Comment by parasti** [comment]
Seems the domain name expired.

The server itself is still up. But clearly not for long.

**Reply by Adacrop-The-Great** [reply]
That is very true. Also, you should join the Nevercord. There's plenty of discussion about this topic. [https://discord.gg/U6dKHdnq8D](url)
